proto/substrait/algebra.proto

```diff
@@ -131,14 +131,38 @@ message ReadRel {
       message ArrowReadOptions {}
       message OrcReadOptions {}
       message DwrfReadOptions {}
+      message TextReadOptions {
+        // The character(s) used to separate fields.  Common values are comma,
+        // tab, and pipe.
+        string field_delimiter = 1;
+        // The maximum number of bytes to read from a single row.  If a row
+        // exceeds this limit the resulting behavior is undefined.
+        uint64 max_block_size = 2;
+        // The character(s) used to quote strings.  Common values are single
+        // and double quotation marks.
+        string quote = 3;
```

Comment:
Can `null_value` be quoted?  E.g. if `null_value` is `NULL` then is `"NULL"` a null or the string `NULL`?

Reply:
Clarified that null_value happens is checked first and then quote removal happens.